src/components/existing-request/existing-request-display.vue

```diff
@@ -123,10 +123,12 @@
                   <v-col cols="12" v-if="isNotPaid">
                     <v-btn block
                            class="button button-blue"
-                           @click="handleButtonClick('RETRY_PAYMENT')">Retry Payment</v-btn>
```

Comment:
This was a hack to display just 2 buttons when the NR is not paid, and it was even more of a hack due to hard-coded actions and action texts. It's a little less hacky now... :/